ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -227,39 +227,26 @@ presubmits:
     branches:
     - master
     cluster: api.ci
-    context: ci/prow/e2e-aws-scaleup-rhel7
+    context: ci/prow/e2e-aws-upgrade
```

Comment:
I'm not sure what happened, but why did this job get moved up here instead of remaining on line 314?  I think this should be the workers-rhel7 job, that would probably reduce the amount of changes in this diff?

Reply:
Running `make jobs` generated all the files in the `jobs/` directory, with the exception of the after the fact modification of `optional: true`.  When the jobs are generated they are sorted alphabetically which caused additional changes because of the job name change.

Reply:
Yup, that explains it :)  Thanks!